articles/templates/articles/topic.html

```diff
@@ -1,16 +1,23 @@
 <div class="container-fluid topic">
-  <div class="row margin-b-2">
-    <div class="col-lg-12">
-      <h1 id="{{root.title}}"> {{ root.title }} </h1>
-      <a href="/new/topic/{{ root.title }}" class="btn btn-default">New Topic</a>
-    </div>
-    <div class="col-lg-12">
-      {% include 'articles/_article_list.html' with rows=child.rows allow_new=True title=root.title %}
-      {% for child in root.children.all %}
-      {% with "articles/topic.html" as filename %}  
-          {% include filename with root=child %}
-          {% endwith %}
-      {% endfor %}
+  <div class="toggle-wrapper">
+    <div class="row margin-b-2">
+      <div class="toggle">
+        <div class="toggle-icon"><i class="fas fa-caret-square-down"></i></div>
+        <div class="title" id="{{root.title}}"> {{ root.title }} </div>
+      </div>
+      <div class="toggle-content">
```

Comment:
Is there something here that makes the toggle-content hidden vs not hidden?

Reply:
yea it's in base.html